sdk/appconfiguration/Azure.Data.AppConfiguration/src/ConfigurationClientOptions.cs

```diff
@@ -18,13 +18,13 @@ public class ConfigurationClientOptions : ClientOptions
         internal const ServiceVersion LatestVersion = ServiceVersion.V1_0;
 
         /// <summary>
-        /// The versions of App Config Service supported by this client library.
+        /// The versions of the App Configuration service supported by this client library.
```

Comment:
Does Azure refer to it as "App Configuration" or "Application Configuration"? Might want to match its text.

Reply:
It's App Configuration - https://docs.microsoft.com/en-us/azure/azure-app-configuration/overview